app/layout/projects/new/form/component.tsx

```diff
@@ -303,14 +311,18 @@ const ProjectForm: React.FC<ProjectFormProps> = () => {
 
                     {PAOptionSelected === 'regular' && (
                       <>
+
                         <CountryRegionSelector
                           country={values.countryId}
                           region={values.adminAreaLevel1Id}
                           subRegion={values.adminAreaLevel2Id}
+                          onClick={scrollDown(planningAreaScrollRef)}
```

Comment:
It only works without void. It may be the ref, that doesn't render first time.

Reply:
@anamontiaga it will trigger each time during rendering. 

`() => scrollDown(planningAreaScrollRef)`

and

` onClick?: () => void;` will work fine and as expected.